backend/apps/project_health/services.py

```
@@ -0,0 +1,216 @@
+    # Last commit date
+    last_commit_days = None
+    try:
+        commits = _github_get(
+            f"/repos/{owner}/{repo_name}/commits?per_page=1", token=token
+        )
+        if commits:
+            date_str = commits[0]["commit"]["committer"]["date"]
+            last_commit_dt = datetime.fromisoformat(date_str.replace("Z", "+00:00"))
+            last_commit_days = (datetime.now(tz=timezone.utc) - last_commit_dt).days
+    except Exception:
+        pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)